app/views/works/form.html.erb

```diff
@@ -41,8 +41,8 @@
     <% end %>
 
     <% component.with_pane(tab_name: :abstract, label: t('works.edit.panes.abstract.label'), form:) do %>
-      <p><%= t('works.edit.fields.abstract.help_text') %></p>
-      <%= render Elements::Forms::TextareaFieldComponent.new(form:, field_name: :abstract, label: t('works.edit.fields.abstract.label')) %>
+      <p style="width: 700px;"><%= t('works.edit.fields.abstract.help_text') %></p>
```

Comment:
If all help texts will use the same width, I wonder if we might want, e.g., a `help-text` class? (If not, ignore!)

Reply:
I don't understand why we're hard coding widths all over the place. If we're concerned about the readability, we should limit the width of the page, not individual text.

Reply:
I can back out the widths and we can discuss.